src/components/AgreementSections/FormInfo.js

```diff
@@ -67,6 +73,13 @@ export default class FormInfo extends React.Component {
     return null;
   }
 
+  validateUniqueProperty = (value, _, meta) => {
```

Comment:
Technically this isn't strictly to do with uniqueness right? You only know it's uniqueness bc you know that that's the only validation the backend has implemented. Maybe something more generic just talking about backend validation.